dotnet/src/Connectors/Connectors.MistralAI/MistralAIToolCallBehavior.cs

```diff
@@ -179,16 +180,15 @@ internal override void ConfigureRequest(Kernel? kernel, ChatCompletionRequest re
                 return;
             }
 
-            bool autoInvoke = base.MaximumAutoInvokeAttempts > 0;
-
             // If auto-invocation is specified, we need a kernel to be able to invoke the functions.
             // Lack of a kernel is fatal: we don't want to tell the model we can handle the functions
             // and then fail to do so, so we fail before we get to that point. This is an error
             // on the consumers behalf: if they specify auto-invocation with any functions, they must
             // specify the kernel and the kernel must contain those functions.
+            bool autoInvoke = base.MaximumAutoInvokeAttempts > 0;
             if (autoInvoke && kernel is null)
             {
-                throw new KernelException($"Auto-invocation with {nameof(EnabledFunctions)} is not supported when no kernel is provided.");
+                throw new KernelException($"Auto-invocation with {nameof(RequiredFunctions)} is not supported when no kernel is provided.");
```

Comment:
Here, for exception message we use `RequiredFunctions`, but in `ToString` override we use `AnyFunction`, which looks a little bit inconsistent. Would it be possible to use only one name?